example/src/Substrate.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component } from 'react';
+import QrSigner from '@parity/qr-signer';
+import extrinsics from '@polkadot/api-metadata/extrinsics/static';
+import { ImmortalEra, Method, SignaturePayload } from '@polkadot/types';
+import { hexToU8a } from '@polkadot/util';
+
+// Fetch some fake metadata
+Method.injectMethods(extrinsics);
+
+// Create some fake extrinsic
+const balancesTransfer =
+  '0x' +
+  '2502' + // length
+  '81' + // signed flag
+  'ffd172a74cda4c865912c32ba0a80a57ae69abae410e5ccb59dee84e2f4432db4f' + // who
+  'fa4c192f6960a3bcdbdee5bcd9c26a3f971b131081912abcc31eab6a0b7589ab' + // sig1
+  '7f99b81a01738cb5e2a911a19d5daa5c0b654d4b8dbc521a6b29090c6d205903' + // sig2
+  '0000' + // nonce
+  '0500' + // balances.transfer
+  'ffd7568e5f0a7eda67a82691ff379ac4bba4f9c9b859fe779b5d46363b61ad2db9' + // to
+  'e56c'; // value
+
+// Create signature payload of the above extrinsic
+const signaturePayload = new SignaturePayload({
+  nonce: 1,
+  method: balancesTransfer,
+  era: new ImmortalEra(),
+  blockHash:
+    '0x6f6f9bba0eed8e3ae9446c37eee763f93118b52a315a7b46090453ba6288da1f'
+});
+
+export default class Substrate extends Component {
+  state = {
+    scan: false,
+    scanned: null
+  };
+
+  render() {
+    const { scan, scanned } = this.state;
+
+    return (
+      <div>
+        <h2>Substrate</h2>
+        {scanned ? (
+          <div>Substrate Scanned: {JSON.stringify(scanned)}</div>
+        ) : (
+          <QrSigner
+            size={300}
+            scan={scan}
+            network="substrate"
+            payload={{
+              account: hexToU8a(
+                '0xd7568e5f0a7eda67a82691ff379ac4bba4f9c9b859fe779b5d46363b61ad2db9'
+              ),
+              action: 'signTransaction',
+              crypto: 'ed25519',
```

Comment:
I see this in the spec as well, but really dunno how it is relevant. So, I scan the account, the dapp that has that address couldn't care less if it is ed25519/sr25519 or something else. The signer knows and manages the keys.

So basically, the dapp will only send the data through, the mobile signer can sign it with whichever crypto it has created the account with.